src/main/resources/locales/en/account-home.json

```diff
@@ -15,7 +15,7 @@
     {
       "title": "Single Justice Procedure cases",
       "description": "A nationwide list of cases, ready to be decided by a magistrate without a hearing. Such as TV licensing prosecutions.",
-      "link": "single-justice-procedure"
+      "link": "publication?courtId=0"
```

Comment:
the link can live in the nunjuks file as this locales is for when the language switches from english to welsh, and the link will be the same regardless of the language